client/src/constants.ts

```diff
@@ -28,3 +32,16 @@ export const UPDATE_FREQUENCY_LABELS: { [K in UpdateFrequency]: string } = {
   monthly: "Mensuelle (ou plusieures fois pas mois)",
   yearly: "Annuel (ou plusieures fois par an)",
 };
+
+export const GEOGRAPHICAL_COVERAGE_LABELS: {
+  [K in GeographicalCoverage]: string;
+} = {
+  municipality: "Communale",
+  EPCI: "epci",
```

Comment:
Oupsie c'était l'inverse (les clés correspondent aux valeurs de l'enum renvoyées par l'API, et les valeurs sont les labels)

```suggestion
  epci: "EPCI",
```